group_project/assignment_6.py

```diff
@@ -7,9 +7,10 @@
 O  |  X  |  O
 """
 
-
 def format_tic_tac_toe_board(first_row, second_row, third_row):
-    pass
+    board1 = '\n{}  |  {}  |  {}\n--------------\n{}  |  {}  |  {}\n--------------\n{}  |  {}  |  {}\n'.format(first_row[0],first_row[1],first_row[2],second_row[0],second_row[1],second_row[2],third_row[0],third_row[1],third_row[2])
```

Comment:
This line is WAAAAY too long. You can break it using parentheses: https://repl.it/Md46

☝️ this is a good topic to ask and talk about class btw, please remind me.

Reply:
i agree with santiago, it makes it harder to read.